db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> None:
```

Comment:
does this function return `None`? Fix  in all places

Reply:
i set up models.Charfield, if its also incorrect - help me pls)